ssc/CMakeLists.txt

```diff
@@ -197,10 +197,10 @@ if(UNIX AND NOT SAMAPI_EXPORT)
 	if (NOT APPLE)
 		target_link_libraries(dltest -ldl -Wl,--no-as-needed)
 	endif()
-	add_custom_command(
-		TARGET ssc
-		POST_BUILD
-		COMMAND dltest $<TARGET_FILE:ssc>)
+#	add_custom_command(
+#		TARGET ssc
+#		POST_BUILD
+#		COMMAND dltest $<TARGET_FILE:ssc>)
```

Comment:
why are we no longer running this?

Reply:
There was an issue with the build path, adding back now...